src/file/metadata.rs

```diff
@@ -386,4 +474,247 @@ impl ColumnChunkMetaData {
     };
     Ok(result)
   }
+
+  /// Method to convert to Thrift.
+  pub fn to_thrift(&self) -> ColumnChunk {
+    let column_metadata = ColumnMetaData {
+      type_: self.column_type.into(),
+      encodings: self.encodings().into_iter().map(|&v| v.into()).collect(),
+      path_in_schema: Vec::from(self.column_path.as_slice()),
+      codec: self.compression.into(),
+      num_values: self.num_values,
+      total_uncompressed_size: self.total_uncompressed_size,
+      total_compressed_size: self.total_compressed_size,
+      key_value_metadata: None,
+      data_page_offset: self.data_page_offset,
+      index_page_offset: None,
```

Comment:
We convert `index_page_offset` and `statistics` from thrift but not to thrift. Why is that?

Reply:
I think I forgot to add them. Statistics are not currently supported, that is why I passed `None`. But I will still add the code for it here. Will fix.